services/device-registry/src/main/fabric8/deployment.yml

```diff
@@ -23,6 +23,20 @@ spec:
         - sh
         - -c
         - 'cp -u /tmp/hono/example-credentials.json /var/lib/hono/device-registry/credentials.json'
+      # we use another init container to populate the registry
+      # with some default tenants
+      - name: copy-example-tenants
+        image: busybox
+        volumeMounts:
+        - name: conf
+          mountPath: /tmp/hono/example-tenants.json
+          subPath: example-tenants.json
+        - name: registry
+          mountPath: /var/lib/hono/device-registry
+        command:
+        - sh
+        - -c
+        - 'cp -u /tmp/hono/example-tenants.json /var/lib/hono/device-registry/tenants.json'
```

Comment:
why should we use another init container? isn't our setup already complex enough ;-)
seriously, is there a compelling reason to do this?

Reply:
:-))
The truth is : it didn't work by copying the tenants file together with the credentials file (for so far unknown reasons), and this strategy DID work.
Will dig deeper into it and see if I can get rid of the second init container again.